src/commands/project/get-display-name.ts

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable @typescript-eslint/no-use-before-define */
+/* eslint-disable @typescript-eslint/camelcase */
+import commander, { CommandOptions } from "commander";
+import * as fs from "fs";
+import { read as readBedrockYaml } from "../../lib/bedrockYaml";
+import { build as buildCmd, exit as exitCmd } from "../../lib/commandBuilder";
+import { logger } from "../../logger";
+import decorator from "./get-display-name.decorator.json";
+
+/**
+ * Executes the command, can all exit function with 0 or 1
+ * when command completed successfully or failed respectively.
+ *
+ * @param opts validated option values
+ * @param exitFn exit function
+ */
+export const execute = async (
+  opts: CommandOptions,
+  exitFn: (status: number) => Promise<void>
+): Promise<void> => {
+  let currentDirectory = process.cwd() + "/bedrock.yaml";
+  try {
+    // First, find a bedrock.yaml file in this directory and if not, iterate by
+    // one directory up.
+    while (!fs.existsSync(currentDirectory)) {
+      const split = currentDirectory.split("/");
+      if (split.length < 2) {
+        break;
+      }
+      currentDirectory = currentDirectory.replace(
+        split[split.length - 2] + "/",
+        ""
+      );
+    }
+    currentDirectory = currentDirectory.replace("/bedrock.yaml", "");
+    const bedrockFile = readBedrockYaml(currentDirectory);
+    const servicePath = process.cwd().replace(currentDirectory, ".");
+    for (const serviceIndex in bedrockFile.services) {
+      if (servicePath === bedrockFile.services[serviceIndex].path) {
+        console.log(bedrockFile.services[serviceIndex].displayName);
+        await exitFn(0);
+      }
+    }
+    // Ideally, we should not be getting here, so throw an error.
+    throw new Error("Display name for current directory could not be found.");
```

Comment:
don't need `new`. Also we're moving our thrown errors into a new pattern. An example: https://github.com/CatalystCode/spk/pull/505#pullrequestreview-387384084